server/app/controllers/FileController.java

```diff
@@ -82,7 +76,7 @@ public CompletionStage<Result> show(Request request, long applicantId, String fi
    */
```

Comment:
Nit: change time -> team while we're here.

Reply:
Actually, is this comment referring to the old code we deleted here? If so, let's remove?

Reply:
It does refer to the old code, but we need to keep it around to continue supporting the old URL pattern, even though the auth logic is no longer divergent.